### PR TITLE
Nex is good at cocket league

### PR DESCRIPTION
**Pull request by NexInfinite**

### CODE REVIEW

**Comment by auguwu** [comment]
cock and ball torture

**Comment by NexInfinite** [comment]
> cock and ball torture

Yeah that checks out. Speaking of checking out, Tuesday?

File: cmds/setplaying.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict';
-
+// Nex is a god at rl
```
Comment (AlekEagle) [inline cmds/setplaying.js:2]: please be truthful

**Review by AlekEagle — CHANGES_REQUESTED** [review]

File: cmds/setplaying.js (hunk 1 above)
Comment (NexInfinite) [reply cmds/setplaying.js:2]: I am

**Pull request merged**